internal/envoy/v3/listener.go

```diff
@@ -50,10 +51,11 @@ const (
 	HTTPVersion2    HTTPVersionType = http.HttpConnectionManager_HTTP2
 	HTTPVersion3    HTTPVersionType = http.HttpConnectionManager_HTTP3
 
-	HTTPFilterRouter  = "type.googleapis.com/envoy.extensions.filters.http.router.v3.Router"
-	HTTPFilterCORS    = "type.googleapis.com/envoy.extensions.filters.http.cors.v3.Cors"
-	HTTPFilterGrpcWeb = "type.googleapis.com/envoy.extensions.filters.http.grpc_web.v3.GrpcWeb"
-	HTTPFilterGzip    = "type.googleapis.com/envoy.extensions.compression.gzip.compressor.v3.Gzip"
+	HTTPFilterRouter    = "type.googleapis.com/envoy.extensions.filters.http.router.v3.Router"
+	HTTPFilterCORS      = "type.googleapis.com/envoy.extensions.filters.http.cors.v3.Cors"
+	HTTPFilterGrpcStats = "type.googleapis.com/envoy.extensions.filters.http.grpc_stats.v3.FilterConfig"
```

Comment:
do we need this? doesn't look like it is used anywhere?

Reply:
Removed.